GitUI/GitUICommands.cs

```diff
@@ -1487,6 +1487,12 @@ private bool RunCommandBasedOnArgument(IReadOnlyList<string> args, IReadOnlyDict
                         return StartCloneDialog(null, args[1].Replace("github-mac://openRepo/", ""), true);
                     }
 
+                    // User supplied `.` as param. So, we need to open the current working directory.
+                    if (args[1] == ".")
+                    {
+                        return StartBrowseDialog();
+                    }
```

Comment:
If we start to support ".", why not any path? Thoughts, @gitextensions/git-extensions-team?

Then `Program.GetWorkingDir` should try `args[1]`, too. Currently, it is hard-coded to `args[2]`.

Reply:
Why not?

Reply:
I'm with @mstv - "." denotes a path, so its handling should be generic to any path.

Reply:
So, do we need to create a separate PR for that? Or do I need to make changes to this one?
CC: @mstv

Reply:
Anyway, I started working on it.

Reply:
Implemented it.